pennylane/gradients/metric_tensor.py

```diff
@@ -190,6 +190,33 @@ def metric_tensor(  # pylint:disable=too-many-arguments, too-many-positional-arg
         a warning is raised and the block-diagonal approximation is computed instead.
         It is significantly cheaper in this case to explicitly set ``approx="block-diag"`` .
 
+    .. note::
+
+        When used with Catalyst, the classical component of the circuit is not included.
+        This matches the results of setting ``hybrid=False``.
+
+        For example,
+
+        >>> from jax import numpy as jnp
+        >>> @qml.qnode(qml.device('lightning.qubit', wires=4))
+        ... def c(x, y):
+        ...     qml.RX(2*x, 0)
+        ...     qml.RY(y, 0)
+        ...     return qml.expval(qml.Z(0))
```

Comment:
```suggestion
        ...     return qml.expval(qml.Z(0))

```
For compatibility of copy-paste in REPL env

Reply:
But wouldn't that break everything into two blocks?

Reply:
and it seems to copy-paste fine to me.

Reply:
Hmmm interesting here's what I got:
```bash
(venv) ➜  pennylane git:(shots-decoupling/helpers) ✗ python
Python 3.11.12 (main, Apr  9 2025, 08:55:54) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from jax import numpy as jnp
>>> @qml.qnode(qml.device('lightning.qubit', wires=4))
... def c(x, y):
...     qml.RX(2*x, 0)
...     qml.RY(y, 0)
...     return qml.expval(qml.Z(0))
... qml.qjit(qml.metric_tensor(c))(jnp.array(0.5), jnp.array(0.6))
  File "<stdin>", line 6
    qml.qjit(qml.metric_tensor(c))(jnp.array(0.5), jnp.array(0.6))
    ^^^
SyntaxError: invalid syntax
>>> qml.metric_tensor(c, argnums=(0,1))(jnp.array(0.5), jnp.array(0.6))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'c' is not defined
>>> qml.metric_tensor(c, hybrid=False)(qml.numpy.array(0.5), qml.numpy.array(0.6))
```